packages/atomic/src/templates/default.html

```diff
@@ -1,8 +1,6 @@
 <style>
-  :host {
-    border-top: 1px solid var(--atomic-neutral);
+  :host > .result-root {
     display: block;
-    padding: 15px;
```

Comment:
I removed the old divider and padding from the default template. Here what the default template looks like with the new dividers and padding:
<details>
<summary>Preview</summary>

![image](https://user-images.githubusercontent.com/54454747/122592620-839d5300-d032-11eb-8f18-ca1767a02a26.png)
</details>

Obviously, this default template is unaffected by density changes and such.

Reply:
Resolving this comment since I replaced the default template with just a title.